File: src/kyu_7/alphabet_war/js/alphabetWarTest.js
const Test = require('@codewars/test-compat');
const alphabetWar = require("./alphabetWar");

describe("Tests", () => {
    it("test", () => {
        Test.assertEquals( alphabetWar("z") , "Right side wins!" );
        Test.assertEquals( alphabetWar("zdqmwpbs") , "Let's fight again!" );
        Test.assertEquals( alphabetWar("zzzzs"), "Right side wins!" );
        Test.assertEquals( alphabetWar("wwwwww"), "Left side wins!" );
    });
});

File: src/kyu_7/alphabet_war/java/AlphabetWarTest.java
package kyu_7.alphabet_war.java;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class AlphabetWarTest {
    @Test
    public void BasicTest() {
        assertEquals("Right side wins!", Kata.alphabetWar("z"));
        assertEquals("Let's fight again!", Kata.alphabetWar("zdqmwpbs"));
        assertEquals("Right side wins!", Kata.alphabetWar("zzzzs"));
        assertEquals("Left side wins!", Kata.alphabetWar("wwwwwwz"));
    }
}